src/ripple/protocol/Feature.h

```diff
@@ -369,7 +369,7 @@ extern uint256 const featurePayChan;
 extern uint256 const featureFlow;
 extern uint256 const featureCompareTakerFlowCross;
 extern uint256 const featureFlowCross;
-extern uint256 const featureCryptoConditions;
+extern uint256 const retiredCryptoConditions;
```

Comment:
I noticed no rule enabled calls were removed. I assume the amendment was never proposed?

Reply:
I'm not entirely sure about the history.  Apparently the amendment was already unconditionalized before I got here.  It looks like it had to do with `SusPay`, which was never enabled (https://xrpl.org/known-amendments.html#cryptoconditions).  But the CryptoConditions amendment is live and on the ledger.